.github/workflows/linux.yml

```
@@ -9,6 +9,16 @@ on:
     - v[0-9]+.[0-9]+.[0-9]+
   pull_request:
     types: [opened, synchronize, reopened, ready_for_review]
+    paths-ignore:
+      - ".github/ISSUE_TEMPLATE/**"
+      - ".github/.jira_sync_config.yaml"
+      - ".github/workflows/automatic-doc-checks.yml"
+      - ".github/workflows/cla-check.yaml"
+      - "debian/**"
+      - "docs/**"
+      - "README.md"
+      - "renovate.json5"
+      - "SECURITY.md"
```

```suggestion
      - "SECURITY.md"
      - "LICENSE"
      - ".gitignore"
      - ".clang-format"
```

A few more if you're interested.

---

I considered those too, but here's why I didn't include them:

- I think we include LICENSE in the package. It changes very infrequently anyway, so I didn't bother to confirm. I can include it if you're sure that is not the case.
- changes to `.gitignore` could cause builds to fail (for example, if they cause necessary source files to be uncommitted by mistake)
- changes to `.clang-format` could cause the Lint step to fail

Does this reasoning make sense to you?